yellowstone-grpc-tools/src/scylladb/sink.rs

```diff
@@ -256,26 +257,16 @@ impl Shard {
                 if offset % SHARD_OFFSET_MODULO == 0 && offset > 0 {
                     // Make sure the last period is committed
                     let t = Instant::now();
-                    let result = self
-                        .session
-                        .execute(
-                            &commit_period_ps,
-                            (curr_period - 1, producer_id, shard_id, curr_period - 2),
-                        )
-                        .await?
-                        .first_row_typed::<(bool, ShardPeriod)>();
-                    let (lwt_success, last_committed_period) = result?;
-                    if lwt_success || last_committed_period == (curr_period - 1) {
-                        info!(
-                            "shard={},producer_id={:?} committed period: {}: time to commit: {:?}",
-                            shard_id,
-                            self.producer_id,
-                            curr_period,
-                            t.elapsed()
-                        );
-                    } else {
-                        anyhow::bail!("Period committment failed for period {}", curr_period - 1);
-                    }
+                    self.session
+                        .execute(&commit_period_ps, (producer_id, shard_id, curr_period - 1))
+                        .await?;
+                    info!(
+                        "shard={},producer_id={:?} committed period: {}: time to commit: {:?}",
+                        shard_id,
+                        self.producer_id,
+                        curr_period,
+                        t.elapsed()
+                    );
```

Comment:
```suggestion
                    info!(
                        shard = shard_id,
                        producer_id = ?self.producer_id,
                        committed_period = curr_period,
                        time_to_commit = ?t.elapsed()
                    );
```

with tracing it's possible to use fields, the benefit of this is that if the output is json then getting field values is much easier
no need to change, just an example of tracing feature

Reply:
much nicer indeed!